src/variables.ts

```diff
@@ -1,8 +1,23 @@
-interface Idata {
-  navbarLinks: string[],
-  filterBlockTabs: string[]
+export interface Input {
+  name: string;
+  type: string;
+  label: string;
+  labelId: string;
+  regExp?: { [Symbol.match](string: string): RegExpMatchArray | null };
+  isValidated: boolean;
+}
 
+interface Idata {
```

Comment:
Please avoid using `I` in interfaces naming.

Reply:
renamed, thanks